specs/experimental/builder-api/sequencer.md

```diff
@@ -0,0 +1,205 @@
+# Honest Sequencer
+
+**Notice**: This document is a work-in-progress for researchers and implementers.
+
+## Table of contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Constants](#constants)
+- [Sequencer registration](#sequencer-registration)
+  - [Preparing a registration](#preparing-a-registration)
+  - [Signing and submitting a registration](#signing-and-submitting-a-registration)
+  - [Registration dissemination](#registration-dissemination)
+- [OP chain responsibilities](#op-chain-responsibilities)
+  - [Block proposal](#block-proposal)
+    - [Constructing the `BeaconBlockBody`](#constructing-the-opblockbody)
+      - [ExecutionPayload](#executionpayload)
+      - [Bid processing](#bid-processing)
+    - [Relation to local block building](#relation-to-local-block-building)
+- [Liveness failsafe](#liveness-failsafe)
+- [How to avoid slashing](#how-to-avoid-slashing)
+  - [Proposer slashing](#proposer-slashing)
+- [Responsibilities during the Merge transition](#responsibilities-during-the-merge-transition)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction
+
+This document explains the way in which an OP Stack sequencer is expected to use the [Builder spec][builder-spec] to participate in an external builder network.
+
+At a high-level, there is a registration step the sequencer must perform ahead of any proposal duties so builders know how to craft blocks for their specific proposal. Having performed the registration, a sequencer waits until it is their turn to propose the next block in the chain. The sequencer then requests an `ExecutionPayload` from the external builder network to put into their `UnsafeBlock` in lieu of one they could build locally.
+
+## Prerequisites
+
+This document assumes knowledge of the terminology, definitions, and other material in the [Builder spec][builder-spec]
+and by extension the [Bellatrix consensus specs][bellatrix-specs].
+
+## Constants
+
+| Name | Value | Units |
+| - | - | - |
+| `EPOCHS_PER_SEQUENCER_REGISTRATION_SUBMISSION` | 1 | epoch(s)|
```

Comment:
Why deal in epochs? It seems to be using L1 consensus terms when it doesn't need to